src/apps/external_executor/protobuf/kv.proto

```diff
@@ -13,8 +13,10 @@ option optimize_for = LITE_RUNTIME;
 service KV
 {
   // Indicate to node that this executor is ready for a new request.
-  // Returns description of a client request, to be executed.
-  rpc StartTx(google.protobuf.Empty) returns (OptionalRequestDescription) {}
+  // Returns stream of descriptions of client requests, to be executed.
+  rpc Activate(google.protobuf.Empty) returns (stream Work) {}
```

Comment:
Could we update [this sequence diagram](https://microsoft.github.io/CCF/main/architecture/request_flow.html#external-executor-flow) accordingly? (doesn't have to be part of this PR).

Reply:
Yes. I'll update this diagram/associated docs once the flow has finalised, but I think its still in flux at the moment and this form is unlikely to stay for long.